src/Symfony/Component/Security/Core/composer.json

```diff
@@ -17,25 +17,25 @@
     ],
     "require": {
         "php": ">=5.3.9",
-        "paragonie/random_compat": "~1.0"
```

Comment:
yep, I'm going to add random_compat to polyfill-php70 that's why it's removed from here